server.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const server = express();
+const bodyParser = require("body-parser")
+
+let mysql = require('mysql')
+
+let connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: '',
+    database: 'users'
+})
+
+connection.connect(function(err){
+    if(err) throw err
+        console.log("You're connected.")  
+})
```

Comment:
what is request comes in before database connection is established?

Reply:
sorry but I didn't understand you. Can you please paraphrase it.